server/service/todoService.js

```diff
@@ -0,0 +1,141 @@
+import TodoModel from "../model/todosModel.js";
+
+class TodoService {
+    async deleteTodo(req, res) {
```

Comment:
just delete, don't need to name it like deleteTodo, because the context is given by the class name